Dockerfile

```diff
@@ -1,8 +1,10 @@
+ARG BASEIMAGE=alpine:3.15.0
```

Comment:
My only bad feeling about this is that we probably can't use dependabot anymore to keep the image up-to-date. But we can't use it for the distroless image anyway, because it's not mentioned in this file anywhere, so there will be manual steps to keep images up-to-date.